pkg/helm/urlfetcher.go

```diff
@@ -0,0 +1,155 @@
+package helm
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"istio.io/istio/pkg/log"
```

Comment:
you should set up goimports to order these for you. generally it's
built-in packages

external packages

istio packages

with blank line between.